### ISSUE TITLE
view function regression in latest version

### ISSUE THREAD

**Issue by ninamillik** [glotzerlab/signac#208]
### Description

"...there appears to be a regression affecting the view function in the latest version." per @csadorf  in the Gitter signac lobby

### To reproduce
```
# init.py
import signac

project = signac.init('fmria_project')

sp = {'monomer':'nfa'}
job = project.open_job(sp)
job.init()
```
`$ python3 init.py`
`$ signac view`

### Error output

`Error: <lambda>() takes 1 positional argument but 2 were given`

### System configuration

Please complete the following information:

 - Operating System [e.g. macOS]: CentOS `6.10 (final)`
 - Version of Python [e.g. 3.7]: `3.7.3`
 - Version of signac [e.g. 1.0]: `1.1.0`

**Comment by bdice** [comment]
Thanks for the bug report, @ninamillik! I can reproduce the problem. `project.create_linked_view('myview')` appears to work for me, so I think this may be a CLI issue.

@ninamillik If you want a quick fix, try running `signac shell -c "project.create_linked_view()"` from the project root directory.

**Comment by ninamillik** [comment]
Thank you! `signac shell -c "project.create_linked_view()"` worked perfectly!

**Comment by csadorf** [comment]
Quick update: It turns out that this issue is actually more related to the fact that the workspace had only one job. We have created a patch (PR #211 ) that should resolve this problem and will be released as of version 1.2.
